### PR TITLE
Added The UI For Dish Screen

### PR DESCRIPTION
**Pull request by KarandeepSingh15**
I Have tried and added the UI Screen for dishes using compose only.Please Review

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
```
Comment (Kardelio) [inline .idea/gradle.xml]: I shouldn't have commited `.idea/.gradle.xml` to the repo, sorry about this but could you `reset` this file (aka remove this change) as that will resolve your conflict and this code doesn't do anything useful for the codebase

File: .idea/misc.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (Kardelio) [inline .idea/misc.xml]: I shouldn't have commited `.idea/.misc.xml` to the repo, sorry about this but could you `reset` this file (aka remove this change) as that will resolve your conflict and this code doesn't do anything useful for the codebase

File: app/build.gradle [hunk 3]
```diff
@@ -32,7 +32,7 @@ android {
         targetCompatibility JavaVersion.VERSION_11
     }
     kotlinOptions {
-        jvmTarget = JavaVersion.VERSION_11
+        jvmTarget = '1.8'
```
Comment (Kardelio) [inline app/build.gradle]: Nope, sorry, VERSION_11 needs to stay. Please could you reset this change

**Comment by Kardelio** [comment]
@KarandeepSingh15 Hey sorry about the delay im really busy at the moment, however i have managed to take a look at this PR...
So overall its fine! There are a few changes (already commented on each) that I would like you to address if thats ok, however the rest is fine! Im really impressed at your great Compose work.
However I was surprised to see that you created an additional activity, BUT its really not too important as this issue only really asked for the Compose to be created and you did that. I will probably (in a future commit) remove the new activity and instead  keep the Compose calls all from within the single MainActivity, using a NavHost (See this change in another branch: https://github.com/Kardelio/Disher/commit/714321c1954b58555761588cd2b59b65aafeff96#diff-9bf65e4ab796d7a22985a4aab3b472990c738139ea40005d98f2c76019ab7b31R42)

But for now im fine with this and you did a great job with (I think) your first proper try at compose! Well done!

if you could just address this issues that I have made in the PR then i will happily merge...

**Review by Kardelio — CHANGES_REQUESTED** [review]
@KarandeepSingh15 Hey sorry about the delay im really busy at the moment, however i have managed to take a look at this PR...
So overall its fine! There are a few changes (already commented on each) that I would like you to address if thats ok, however the rest is fine! Im really impressed at your great Compose work.
However I was surprised to see that you created an additional activity, BUT its really not too important as this issue only really asked for the Compose to be created and you did that. I will probably (in a future commit) remove the new activity and instead keep the Compose calls all from within the single MainActivity, using a NavHost (See this change in another branch: 714321c#diff-9bf65e4ab796d7a22985a4aab3b472990c738139ea40005d98f2c76019ab7b31R42)

But for now im fine with this and you did a great job with (I think) your first proper try at compose! Well done!

if you could just address this issues that I have made in the PR then i will happily merge...

**Comment by KarandeepSingh15** [comment]
> @KarandeepSingh15 Hey sorry about the delay im really busy at the moment, however i have managed to take a look at this PR... So overall its fine! There are a few changes (already commented on each) that I would like you to address if thats ok, however the rest is fine! Im really impressed at your great Compose work. However I was surprised to see that you created an additional activity, BUT its really not too important as this issue only really asked for the Compose to be created and you did that. I will probably (in a future commit) remove the new activity and instead keep the Compose calls all from within the single MainActivity, using a NavHost (See this change in another branch: [714321c#diff-9bf65e4ab796d7a22985a4aab3b472990c738139ea40005d98f2c76019ab7b31R42](https://github.com/Kardelio/Disher/commit/714321c1954b58555761588cd2b59b65aafeff96#diff-9bf65e4ab796d7a22985a4aab3b472990c738139ea40005d98f2c76019ab7b31R42))
> 
> But for now im fine with this and you did a great job with (I think) your first proper try at compose! Well done!
> 
> if you could just address this issues that I have made in the PR then i will happily merge...

Thank You for motivating me to do it using Compose.I have made the stated changes .

**Review by Kardelio — APPROVED** [review]
All Good , well done

**Pull request merged**